test/utils/getters.go

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
getters are probably not the best name for this file.
maybe "conditions.go" or sth like that?